### ISSUE TITLE
[iPadOS] Can't use shortcuts to copy/paste a line in editor

### ISSUE THREAD

**Issue by ManiaciaChao** [microsoft/vscode#141042]
Does this issue occur when all extensions are disabled?: Yes

- Version: 1.63.2
- Commit: 899d46d82c4c95423fb7e10e68eba52050e30ba3

It seems like `vscode` handovers copy/paste/cut to browser. However, action `Copy Line Down/Up` works as expected. 

**Steps to Reproduce:**

1. Open [monaco-editor](https://microsoft.github.io/monaco-editor/) or [vscode.dev](vscode.dev).
2. Create a new file and enter some text.
3. Copy/Cut/Paste without selection:
    3.1. by stroking `cmd + c/x/v` with an external keyboard.
    3.2. from `workbench.action.showCommands`.
    3.3. from right-click menu.

**Expected behaviour:**

Copy/Cut/Paste the whole line.

**Actual behaviour:**

Copy/Cut won't work. Paste only works for clipboard item.

**Comment by rebornix** [comment]
@TylerLeonhardt can you please help do some initial verification/reproduce?

**Comment by TylerLeonhardt** [comment]
I can verify that this does happen... I worry that there's nothing we can do about it as the OS is likely handling the cmd+c/v/x keybindings.

**Comment by ManiaciaChao** [comment]
Yes, after two years of usage, there’s not much we can do about it. However, we could consider binding copy/paste to different key combinations and managing the clipboard buffer internally, without interacting with the system clipboard. This approach would be similar to how the Vim extension handles it.

**Comment by TylerLeonhardt** [comment]
Yeah, that's an option. It's tough because keybindings that aren't already taken a few and far between... but maybe iPadOS just has to be different.
